presto-main/src/test/java/com/facebook/presto/operator/aggregation/differentialentropy/TestFixedHistogramMLEAggregation.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.differentialentropy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestFixedHistogramMLEAggregation
+        extends TestFixedHistogramAggregation
+{
+    public TestFixedHistogramMLEAggregation()
+    {
+        super(Constants.FIXED_HISTOGRAM_MLE_METHOD_NAME);
+    }
+
+    @Override
+    public Double getExpectedValue(int start, int length)
+    {
+        List<Double> samples = new ArrayList<Double>();
```

Comment:
new ArrayList<Double>() -> new ArrayList<>()

Reply:
Changed in all cases in the test directory.